tests/exchange/test_exchange.py

```diff
@@ -2919,3 +2919,60 @@ def test_calculate_fee_rate(mocker, default_conf, order, expected) -> None:
 ])
 def test_calculate_backoff(retrycount, max_retries, expected):
     assert calculate_backoff(retrycount, max_retries) == expected
+
+
+@pytest.mark.parametrize("exchange_name", EXCHANGES)
+def test_get_funding_fees(default_conf, mocker, exchange_name):
+    api_mock = MagicMock()
+    api_mock.get_funding_fees = MagicMock(return_value=[
```

Comment:
You're patching the API here (`self._api.xxxxx` when seen from within the exchange) ... your naming should therefore be `fetch_funding_fees` - the ccxt method name.